cards_information.py

```diff
@@ -1,15 +1,13 @@
-import discord
-from discord import app_commands
 from discord.app_commands import Choice
 
 cards_tiers = [
-    Choice(name='Common', value='Common'),
-    Choice(name='Uncommon', value='Uncommon'),
-    Choice(name='Rare', value='Rare'),
-    Choice(name='Epic', value='Epic'),
-    Choice(name='Mythical', value='Mythical'),
-    Choice(name='Legendary', value='Legendary'),
-    ]
+	Choice(name='Common', value='Common'),
+	Choice(name='Uncommon', value='Uncommon'),
+	Choice(name='Rare', value='Rare'),
+	Choice(name='Epic', value='Epic'),
+	Choice(name='Mythical', value='Mythical'),
+	Choice(name='Legendary', value='Legendary'),

```

Comment:
This makes the indentation consistent with the rest of the file, which uses tabs.